packages/wonder-blocks-i18n/src/components/i18n-inline-markup.jsx

```diff
@@ -0,0 +1,179 @@
+// @flow
+
+/**
+ * Render some HTML, using functions passed in as props to render tags.
```

Comment:
Should be able to migrate all of this into a storybook MDX file when I work on docs.